### PR TITLE
Add store and recipe component

### PR DESCRIPTION
**Pull request by NishaVijai**
Test version of redux store

### CODE REVIEW

**Review by mladen-krstin — APPROVED** [review]
Everything seams good, nice work

File: src/features/recipe/recipeSlice.js
Comment (ocsiddisco) [inline src/features/recipe/recipeSlice.js:1]: lign 25: Async/await and .then are two differents approches to handle async function. Try to use only one (async/await is the last version)
 const response = await axios.request(options);
    return response.data;

**Comment by ocsiddisco** [comment]
as discussed on discord, merging is on main instead of development and api key is visible. I do not merge and close the PR.

File: src/constants/apiData.js
Comment (Alexandra2888) [inline src/constants/apiData.js:1]: API key should be in an .env file

**Review by Alexandra2888 — COMMENTED** [review]
I won't merge PR as we discussed on Discord

**Pull request closed without merging**